### PR TITLE
Create button test

### PR DESCRIPTION
**Pull request by ExplosiveDev**

### CODE REVIEW

File: Admin_App/MainWindow.xaml [hunk 1]
```diff
@@ -7,6 +7,6 @@
         mc:Ignorable="d"
         Title="MainWindow" Height="450" Width="800">
     <Grid>
-
+        <Button Click="Button_Click">Click</Button>
```
Comment (temeirro) [inline Admin_App/MainWindow.xaml:10]: top

**Review by temeirro — APPROVED** [review]

File: Admin_App/MainWindow.xaml.cs [hunk 2]
```diff
@@ -15,14 +15,16 @@
 
 namespace Admin_App
 {
-    /// <summary>
-    /// Interaction logic for MainWindow.xaml
-    /// </summary>
```
Comment (VictorLyalchuk) [inline Admin_App/MainWindow.xaml.cs:20]: Why?

File: Admin_App/MainWindow.xaml.cs [hunk 3]
```diff
@@ -15,14 +15,16 @@
...
     public partial class MainWindow : Window
     {
         public MainWindow()
         {
             InitializeComponent();
         }
+
+        private void Button_Click(object sender, RoutedEventArgs e)
```
Comment (VictorLyalchuk) [inline Admin_App/MainWindow.xaml.cs:25]: Super

File: Admin_App/MainWindow.xaml (hunk 1 above)
Comment (ExplosiveDev) [reply Admin_App/MainWindow.xaml:10]: thanks

Comment (VictorLyalchuk) [reply Admin_App/MainWindow.xaml:10]: Good

**Review by VictorLyalchuk — APPROVED** [review]

**Pull request merged**